metals/src/main/scala/scala/meta/internal/metals/ScalaVersions.scala

```diff
@@ -22,9 +22,18 @@ object ScalaVersions {
     }
 
   val isLatestScalaVersion: Set[String] =
-    Set(BuildInfo.scala213, BuildInfo.scala212, BuildInfo.scala211)
-
-  def recommendedVersion(scalaVersion: String): String = BuildInfo.scala212
+    Set(BuildInfo.scala212, BuildInfo.scala213)
+
+  def recommendedVersion(scalaVersion: String): String = {
+    val binaryVersion = scalaBinaryVersionFromFullVersion(scalaVersion)
+    isLatestScalaVersion
+      .find(latest =>
+        binaryVersion == scalaBinaryVersionFromFullVersion(latest)
+      )
+      .getOrElse {
+        BuildInfo.scala212
```

Comment:
Just out of curiosity, if we do fall into this, why do we recommend 12 over 13?

Reply:
Good question, mostly because we are using it Metals I guess, so we are most sure about. We could do the newest available version.

Reply:
I'm sort of on the side that if we are in this scenario, we should recommend 13. /shrug

Reply:
Switched to 2.13